Frameworks/Foundation/_NSInvocation.arm.asm

```diff
@@ -0,0 +1,82 @@
+;******************************************************************************
+;
+; Copyright (c) Microsoft. All rights reserved.
+;
+; This code is licensed under the MIT License (MIT).
+;
+; THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+; IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+; FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+; AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+; LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+; OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+; THE SOFTWARE.
+;
+;******************************************************************************
+
+    AREA |.text|,ALIGN=4,THUMB,CODE,READONLY
+; r0 = arena start
+; r1 = frame
+; r2 = fn
+; r3 = vfp count
+    ALIGN
+|_CallFrameInternal_VFP| PROC
+    GLOBAL _CallFrameInternal_VFP
+    GLOBAL _CallFrameInternal
+    CMP r3, 1
+    VLDREQ.64 d0, [r0]
+    VLDMGT.64 r0, {d0-d7}
```

Comment:
Conditional VFP instructions are deprecated as of ARMv8 and not officially supported on Windows. You should probably prefer to use a branch here.